artist/tests.py

```diff
@@ -191,6 +191,7 @@ def testArtistApplication(self):
                     "We are a really cool heavy metal band. We mostly perform covers but are excited to "
                     "create an album, and we're hoping PerDiem can help us do that."
                 ),
+                "project": "$Album",
```

Comment:
Was the $ sign a typo?

In following the patterns of what I'm seeing on the site, this seems like it should be the name of the album, which based on the line below you could say `Access Granted`:
```suggestion
                "project": "Access Granted"
```